src/common/eeprom.cpp

```diff
@@ -83,7 +81,7 @@ typedef struct _eeprom_vars_t {
     uint8_t FILE_SORT;
     uint8_t MENU_TIMEOUT;
 #if (EEPROM_FEATURES & EEPROM_FEATURE_SHEETS)
-    uint8_t ACTUAL_SHEET;
+    uint8_t CURRENT_SHEET;
```

Comment:
every other variable related to this is called active_sheet, so probably calling this one ACTIVE_SHEET would be more consistent